scripts/colmap2nerf.py

```diff
@@ -389,3 +390,40 @@ def closest_point_2_lines(oa, da, ob, db): # returns point closest to both rays
 	print(f"writing {OUT_PATH}")
 	with open(OUT_PATH, "w") as outfile:
 		json.dump(out, outfile, indent=2)
+
+	if len(args.mask_categories) > 1:
+		# Check if detectron2 is installed. If not, install it.
+		import importlib.util
+		package_name = 'detectron2'
+		spec = importlib.util.find_spec(package_name)
+		if spec is None:
+			input("Detectron2 is not installed. Press enter to install it.")
+			os.system("python -m pip install 'git+https://github.com/facebookresearch/detectron2.git'")
```

Comment:
This might fail for users that have `python3` on macOS/Linux or `py` on Windows. You could use `sys.executable`.